be/src/olap/rowset/segment_v2/rle_page.h

```diff
@@ -86,14 +86,12 @@ class RlePageBuilder : public PageBuilder {
 
     Status add(const uint8_t* vals, size_t* count) override {
         DCHECK(!_finished);
-        DCHECK_EQ(reinterpret_cast<uintptr_t>(vals) & (alignof(CppType) - 1), 0)
-                << "Pointer passed to Add() must be naturally-aligned";
-
-        const CppType* new_vals = reinterpret_cast<const CppType*>(vals);
+        auto new_vals = reinterpret_cast<const CppType*>(vals);
         for (int i = 0; i < *count; ++i) {
-            _rle_encoder->Put(new_vals[i]);
+            CppType value;
+            memcpy(&value, &new_vals[i], SIZE_OF_TYPE);
```

Comment:
Better to add comment about why we use memcpy here.
And I think should add a TODO here to make input vals aligned later for better performace.

Reply:
ok